doozerlib/dblib.py

```diff
@@ -161,28 +324,23 @@ def __enter__(self):
         return self
```

Comment:
I could see these fields of the payload but when I see simpledb there are lot more attributes for a build record whose source I didn't know. So just to avoid changing code at lot more places I tried doing transforming column names (to comply minimal convention) just before I am going to make an record entry into db table.